frontend/talentsearch/src/js/components/ExperienceAccordion/individualExperienceAccordions/AwardAccordion.tsx

```diff
@@ -105,7 +110,14 @@ const AwardAccordion: React.FunctionComponent<AwardExperience> = ({
         </p>
       </div>
       <div data-h2-padding="b(left, l)">
-        <Button color="primary" mode="outline">
+        <Button
+          color="primary"
+          mode="outline"
+          onClick={(event) => {
+            event.preventDefault();
+            navigate(editUrl);
+          }}
+        >
```

Comment:
Since this is doing a navigate, we should use a button-styled Link component instead of an actual Button component. (Button-styled Links are a new tool in our common project! 😁)